migrations/20191218152204_users.ts

```diff
@@ -0,0 +1,14 @@
+import * as Knex from "knex";
+
+
+export async function up(Knex: Knex): Promise<any> {
+    await Knex.schema.createTable('users', table => {
```

Comment:
not sure we need user schema for start. 
We've decided to start from migrations and them move forward with them 
* logging
* workers
* migrations
* running  flow

We have to look at those areas first
I think we have to change this schema with the `workers` schema. This is the one we really need.
 The interface could simple for start:

```
name: 'beta' //worker name will be sent from server where this migration going to be run
created_at
updated_at
endpoint // where worker is hosted. We will call this API to communicate with worker
api_key? // not required will figure out later how we are going to make auth
```

Let's change this schema with those fields, then we will be ready to use it in client side